masakari-controller/controller/masakari_util.py

```diff
@@ -535,8 +569,10 @@ def __init__(self, config_object):
 
         self.nova_client = nova_client.Client(self.NOVA_API_VERSION,
                                               session=self.auth_session,
-                                              connect_retries=api_retries)
+                                              connect_retries=api_retries,
+                                              logger=LOG.logger)
 
+    @log_process_begin_and_end.output_log
```

Comment:
It shouldn't output the log since the password appears in message. or if you really want to output the message here the decorator needs additional codes that can mask the password.